langchain/embeddings/openai.py

```diff
@@ -92,20 +92,18 @@ class OpenAIEmbeddings(BaseModel, Embeddings):
             os.environ["OPENAI_API_KEY"] = "your AzureOpenAI key"
 
             from langchain.embeddings.openai import OpenAIEmbeddings
-            embeddings = OpenAIEmbeddings(model="your-embeddings-deployment-name")
+            embeddings = OpenAIEmbeddings(
+                deployment="your-embeddings-deployment-name",
+                model="your-embeddings-model-name"
+            )
             text = "This is a test query."
             query_result = embeddings.embed_query(text)
 
     """
 
     client: Any  #: :meta private:
     model: str = "text-embedding-ada-002"
-
-    # TODO: deprecate these two in favor of model
-    #  https://community.openai.com/t/api-update-engines-models/18597
-    #  https://github.com/openai/openai-python/issues/132
-    document_model_name: str = "text-embedding-ada-002"
```

Comment:
@hwchase17 technically breaking change, how do we feel about it

Reply:
i think fine